jsonnet/telemeter/client/telemeter-client.libsonnet

```diff
@@ -0,0 +1,11 @@
+(import 'kubernetes/kubernetes.libsonnet') + {
+  _config+:: {
+    namespace: 'default',
```

Comment:
the namespace is already defined in the kubernetes/kubernetes.libsonnet file, so this doesn't need to be here, let's strive to keep configuration local to its use

Reply:
:+1: